docker-for-mac/release-notes.md

```diff
@@ -1,17 +1,68 @@
 ---
-description: Change log / release notes per Stable release
-keywords: Docker Desktop for Mac, stable, release notes
+description: Change log / release notes for Docker Desktop Mac
+keywords: Docker Desktop for Mac, release notes
 redirect_from:
 - /mackit/release-notes/
-title: Docker for Mac Stable release notes
+- /docker-for-mac/edge-release-notes/
+title: Docker for Mac release notes
 toc_min: 1
 toc_max: 2
 ---
 
-This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop Stable releases.
+This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
-For information about Edge releases, see the [Edge release notes](edge-release-notes.md). For Docker Desktop system requirements, see
-[What to know before you install](install.md#what-to-know-before-you-install).
+> **Important**
+>
+> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
+{: .important }
+
+## Docker Desktop Community 2.5.4
```

Comment:
I don't think we need the release notes from the 2.5.[2-4] Edge versions here. We should roll them up into a single entry for 3.0.0 Stable compared with the previous Stable, as usual.